dayrepo/snippets/master_views.py

```diff
@@ -13,4 +15,36 @@ def get(self, request: HttpRequest):
             {"cars": cars},
         )
 
-car_list = CarListView.as_view()
\ No newline at end of file
+car_list = CarListView.as_view()
+
+class CarView(View):
+    
+    def get(self, request):
+        return render(
+            request,
+            "car_post.html",
+            {
+                "form": CarForm,
+            },
+        )
+    
+    def post(self, request: HttpRequest):
+        req = request.POST
+        if req.get("serial_number_top") != "":
+            number = int(
+                str(req.get("serial_number_top"))
+                + str(req.get("serial_number_end"))
+            )
+        else:
+            number = req.get("serial_number_end")
```

Comment:
HTMLを確認しましたが、この場合serial_number_topは必須項目ではない　＝ 入力されない場合は単なる空白になる
はずなので、そもそもこの if 文はいらないかなと思いました。
```suggestion
        number = int(
            str(req.get("serial_number_top"))
            + str(req.get("serial_number_end"))
        )
```

また個人的には、数字で受け取るのであればserial_number_topとserial_number_endにフォームの分ける必要性もないような気がしています。
特に、serial_number_top にserial_number_endの値を入力されてしまうリスクがあるためです。まぁ、一旦そこらへんの判断は任せます〜

Reply:
対応しました。
https://github.com/bob-g12/dayrepo-for-transportation/pull/70/commits/9ccbcc22a5ca68a0cd1e724b0c97e210d3f9f122
システム側でハイフンの考慮がされていることは伝わってほしいのでフォームは分けたいですね。